app/brave_generated_resources.grd

```diff
@@ -398,7 +398,7 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         Maximum number of ads to display
       </message>
       <message name="IDS_SETTINGS_BRAVE_REWARDS_STATE_LEVEL_AD_TARGETING_LABEL" desc="The text label for the setting controlling subdivision-level ad targeting">
-        State-level ad targeting
+        Regional ad relevance
```

Comment:
Would be great if we could also update the description to help translators. Thanks

Reply:
Maybe see description from the brave://rewards copy

Reply:
@tmancey, what do you think about the following:

`The text label for the setting that allows Brave to show ads meant specifically for user's region`

Reply:
@tmancey, did you mean the message below, to make it match `brave://rewards` ?

`<message name="IDS_SETTINGS_BRAVE_REWARDS_STATE_LEVEL_AD_TARGETING_DESC_LABEL" desc="The text label for the description of subdivision-level ad targeting">
        This allows Brave to show you ads meant specifically for your state.`

Reply:
That is correct, thanks. And I do like "The text label for the setting that allows Brave to show ads meant specifically for user's region"